core/tracepoint/return_value_spec.rb

```diff
@@ -0,0 +1,14 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint#return_value' do
+    def test; end
+
+    it 'returns value from :return event' do
+      trace_value = nil
+      TracePoint.new(:return) { |tp| trace_value = tp.return_value}.enable
+      test
+      trace_value.should equal(nil)
```

Comment:
Testing for a specific answer (even just `42`) would be much stronger here.

Reply:
yeah, I have updated the PR.